### PR TITLE
Add Platypus Finance Subgraph (Avax)

### PR DESCRIPTION
**Pull request by nemani**

### CODE REVIEW

File: subgraphs/platypus-finance/notice.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+there is some issue with the generated output for asset template that is causing things to fail.
```
Comment (this-username-is-taken) [inline subgraphs/platypus-finance/notice.md:1]: why not move this to README?

File: subgraphs/platypus-finance/src/common/getters.ts [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+  LiquidityPoolFeeType,
+} from "../common/constants";
+import { exponentToBigDecimal } from "./utils/numbers";
+import { getUsdPrice } from "../prices";
+
+let altPoolsInit = false;
+
+export function initAltPoolTemplates(event: ethereum.Event): void {
+  // Start watching the LiquidityPools
+  // Note: I have no idea what happens if I create a dynamic datasource
```
Comment (this-username-is-taken) [inline subgraphs/platypus-finance/src/common/getters.ts:37]: Did you figure out this one?

**Comment by this-username-is-taken** [comment]
Really good job! Appreciate all the hard work! I will merge in this one and have you make the pool changes in a separate PR.

**Review by this-username-is-taken — APPROVED** [review]

File: subgraphs/platypus-finance/src/common/getters.ts (hunk 2 above)
Comment (nemani) [reply subgraphs/platypus-finance/src/common/getters.ts:37]: its runs all mappings twice

Comment (this-username-is-taken) [reply subgraphs/platypus-finance/src/common/getters.ts:37]: can you add that as a comment so people are aware?

Comment (nemani) [reply subgraphs/platypus-finance/src/common/getters.ts:37]: I'll actually remove the note, there are no duplicate datasources. Maybe I can add this method to the development docs somewhere.

**Pull request merged**